server/utils.go

```diff
@@ -1,97 +1,11 @@
 package main
 
 import (
-	"fmt"
 	"net/url"
-	"regexp"
 
 	"github.com/andygrunwald/go-jira"
-	"github.com/pkg/errors"
-
-	"github.com/mattermost/mattermost-server/model"
 )
 
-func (p *Plugin) CreateBotDMPost(userId, message, postType string) (returnErr error) {
```

Comment:
Are all of these changes related to replacing the S3 icon?

Reply:
No, but this function used thee icon, etc.